src/Emitter.ts

```diff
@@ -0,0 +1,38 @@
+import { EventEmitter } from 'events';
+
+/**
+ * Mapping of event names to event handlers.
+ */
+type EventMap = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [name: string]: (...args: any[]) => void;
```

Comment:
I think `any` is appropriate for this case, since this is the general type of an event handler. Also, we override it with a more specific type in this library, and this is really just providing a constraint on the more specific type.